xml/http_cookies.xml

```diff
@@ -49,6 +49,14 @@
     <param pos="0" name="service.product" value="Dynamo"/>
   </fingerprint>
 
+  <fingerprint pattern="^(Bugzilla_login_request_cookie)=.*">
+    <description>Bugzilla</description>
```

Comment:
Here and elsewhere: please provide an example value for the tests.

Reply:
I didn't want to break the "code style", but I agree that it looks useless to me.